web/src/activity_ui.ts

```diff
@@ -59,6 +59,18 @@ export function clear_for_testing(): void {
     user_filter = undefined;
 }
 
+export function update_presence_indicators(): void {
+    $("[data-presence-indicator-user-id]").each(function () {
+        const user_id = Number.parseInt(this.dataset.presenceIndicatorUserId ?? "", 10);
+        assert(!Number.isNaN(user_id));
+        const user_circle_class = buddy_data.get_user_circle_class(user_id);
+        this.classList.remove("user_circle_empty");
+        this.classList.remove("user_circle_green");
+        this.classList.remove("user_circle_idle");
+        this.classList.add(user_circle_class);
```

Comment:
This should use jQuery, and in particular `.attr("data-presence-indicator-user-id")`, like the rest of the codebase.

(`this.dataSet` reformatting the data attribute name is really annoying for `git grep`).

Reply:
@timabbott Changed to jQuery